### PR TITLE
[task_identities] Call sh unify/affiliate after loading identites file

### PR DESCRIPTION
**Pull request by valeriocos**
This code makes sure to call the unify operation of SortingHat when loading an identity file. Furthermore,  also the SH affiliate operation is called at the same time, when the affiliate
param is set to True.

### CODE REVIEW

**Comment by valeriocos** [comment]
No, the `unify` is called also later on: https://github.com/chaoss/grimoirelab-sirmordred/blob/master/sirmordred/task_identities.py#L545 (TaskIdentitiesMerge)

**Comment by sduenas** [comment]
> No, the `unify` is called also later on: https://github.com/chaoss/grimoirelab-sirmordred/blob/master/sirmordred/task_identities.py#L545 (TaskIdentitiesMerge)

Ok but can you explain how this is related to #360 and why this PR fixes it? The problem is relation to affiliation. Unify does not set affiliation information.

**Comment by valeriocos** [comment]
This PR tackles the following part of the issue #360:
> When identities got reset, they should be unified before any enrichment+refresh phase is executed. Between the load of the identities file and the end of the unify command neither enrichment or refresh should be executed.

The bug was introduced here: https://github.com/chaoss/grimoirelab-sirmordred/pull/336 ([details](https://github.com/chaoss/grimoirelab-sirmordred/commit/d7f5bf6dcb43a92291297c6df22ce7daf4a7bf19#diff-6423046309d2cc0c5e1ca2a52c6e49a3R334)) 

I can check the other part (see below), but it may be a different bug.
> One of our production instances with the parameter reset_on_load set to true shown a strange behavior and suddenly all the affiliation entries were set to Unknown. This is not the expected behavior.

**Comment by sduenas** [comment]
> This PR tackles the following part of the issue #360:
> 
> > When identities got reset, they should be unified before any enrichment+refresh phase is executed. Between the load of the identities file and the end of the unify command neither enrichment or refresh should be executed.
> 
> The bug was introduced here: #336 ([details](https://github.com/chaoss/grimoirelab-sirmordred/commit/d7f5bf6dcb43a92291297c6df22ce7daf4a7bf19#diff-6423046309d2cc0c5e1ca2a52c6e49a3R334))
> 
> I can check the other part (see below), but it may be a different bug.
> 
> > One of our production instances with the parameter reset_on_load set to true shown a strange behavior and suddenly all the affiliation entries were set to Unknown. This is not the expected behavior.

Ok. Then I think you can merge this PR but I think I won't fix the other problems. You should check that behaviour and if you don't find it close the issue.

File: sirmordred/task_identities.py [hunk 1]
```diff
@@ -333,6 +333,13 @@ def load_grimoirelab_identities(config):
                 logger.debug("Doing unify after identities load")
                 self.__execute_command(ucmd)
 
+            if 'affiliate' in cfg['sortinghat'] and cfg['sortinghat']['affiliate']:
+                cmd = ['sortinghat', '-u', self.db_user, '-p', self.db_password,
+                       '--host', self.db_host, '-d', self.db_sh]
+                cmd += ['affiliate']
+                logger.debug("Doing unify after identities load")
```
Comment (sanacl) [inline sirmordred/task_identities.py]: Please replace `unify` with `affiliate`

**Comment by sanacl** [comment]
@valeriocos @sduenas LGTM

**Pull request merged**